.travis.yml

```diff
@@ -4,15 +4,18 @@ python:
   - "3.6"
 
 install:
-  - pip install -r requirements.txt
-  - pip install codecov pytest pytest-cov
+  - pip install --upgrade pip
+  - pip install "pytest>=3.6"
+  - pip install codecov pytest-cov pytest-asyncio pylint
+  - python setup.py develop
 
 script:
-  - pytest --cov=./ -v
-  - pylint --rcfile=.pylintrc encryption host libp2p network peer stream_muxer transport tests
+  - pytest --cov=./libp2p tests/
+  - pylint --rcfile=.pylintrc libp2p/!(kademlia|protocol_muxer) tests
```

Comment:
Good catch; we actually only wanted to exclude `kademlia` (it's an external implementation that we haven't used yet; we will likely move to a Git submodule soon (#97)). We can include `protocol_muxer` in linting after #106 lands.

Reply:
just approved #106